.github/workflows/Explorer-for-Endevor-CI.yml

```diff
@@ -0,0 +1,47 @@
+name: Explorer for Endevor CI
+
+on:
+  push:
+     branches:
+      - master
+      - development
+  pull_request:
+    branches:
+      - master
+      - development
+
+jobs:
+  build:
+    runs-on: ${{ matrix.os }}
+
+    strategy:
+      # Continue to run tests on the other systems if one fails
+      fail-fast: false
+      matrix:
+        node-version: [10.x, 12.x]
+        # order operating systems from best to worst
+        os: [windows-latest, ubuntu-latest, macos-latest]
+
+    steps:
+      - name: Check out code
+        uses: actions/checkout@v2
+
+      - name: Use Node.js ${{ matrix.node-version }}
+        uses: actions/setup-node@v1
+        with:
+          node-version: ${{ matrix.node-version }}
+
+      - run: npm install
+
+      - run: npm run vscode:prepublish
```

Comment:
I think this should be `npm run compile` for the time being. I know it's executing the same command, but we can be verbose about the action